src/models/tools/table/table-content.ts

```diff
@@ -35,22 +37,29 @@ export function defaultTableContent(props?: IDefaultContentOptions) {
 }
 
 interface IGetTableContentHeight {
-  dataRows: number;
-  rowHeight?: number;
+  rows: TRow[];
+  rowHeight: (args: RowHeightArgs<TRow>) => number;
+  headerHeight: () => number;
+  getTitleHeight: () => number;
   readOnly?: boolean;
   hasExpressions?: boolean;
   padding?: number;
 }
 export const getTableContentHeight = ({
```

Comment:
Not your code, but it is weird that this function is in the model. I understand it is in the gray area since we are saving the CLUE Document rowHeights in the model. But it seems it would be better for the component to figure out it needs more height based on the browser sizing instead of trying to duplicate the browser sizing based on just the information in the model.